app/models/ride.py

```diff
@@ -0,0 +1,110 @@
+import re
+import datetime
+from ..db.db import *
+
+
+class CheckRideData:
```

Comment:
This class seems to have been repeated in every file, did you consider reusing the original class in your other files?

Reply:
Yes but i thought i would have different classed for verification  so that if an error occurred in one part of my program  then it  would not affect another. But now considering code re-usability it would have been better